meinberlin/assets/scss/components/_formset.scss

```diff
@@ -24,7 +24,13 @@
 }
 
 .formset__delete {
-    display: none;
+    @include sr-only();
+
+    &:focus ~ .formset__delete-label {
+        outline-width: 2px;
+        outline-style: solid;
+        outline-color: Highlight;
+    }
```

Comment:
AFAIK focus styles vary wildly across browsers. Also, the `Highlight` keyword is listed as deprecated on [mdn](https://developer.mozilla.org/en-US/docs/Web/CSS/color).

I guess you took this from some article (e.g. like [this one](https://ghinda.net/article/mimic-native-focus-css/)), but it should be clearly marked as a hack.